ext/adaptors/ranges-lib.tex

```diff
@@ -212,7 +214,8 @@
     @\removed{constexpr bool operator!() const requires ForwardRange<const D>;}@
 
     @\added{template <RandomAccessRange R = const D>}@
-      @\added{constexpr auto data() const requires is_pointer_v<iterator_t<R>>;}@
+        @\added{requires is_pointer_v<iterator_t<R>{>}}@
+      @\added{constexpr auto data() const;}@
```

Comment:
I'm just curious, why?

Reply:
Simply for consistency: we have not used trailing requires clauses on function templates before.